index.js

```diff
@@ -87,6 +87,12 @@ const speakers = [
     quote: 'Member of the Board of Management of BMW AG, Production',
     image: 'assits/speaker3.jpg',
   },
+  {
+    name: 'DR. MILAN NEDELJKOVIĆ',
+    about: 'Production',
+    quote: 'Member of the Board of Management of BMW AG, Production',
+    image: 'assits/speaker3.jpg',
+  },
 ];
```

Comment:
- [ ] On the mobile section, the "More" button does not render other speakers. Kindly see to this 

## view:
![image](https://user-images.githubusercontent.com/50990925/227096421-9e522987-2636-459b-a83a-19d9a78dfdf6.png)

Reply:
done 😁